superset/dashboards/schemas.py

```diff
@@ -159,6 +159,7 @@ class DashboardGetResponseSchema(Schema):
     owners = fields.List(fields.Nested(UserSchema))
     roles = fields.List(fields.Nested(RolesSchema))
     table_names = fields.String()  # legacy nonsense
+    changed_on_humanized = fields.String(data_key="changed_on_delta_humanized")
```

Comment:
1. `superset.models.helps.AuditMixinNullable.changed_on_delta_humanized` is a instance method, unable to serializes.
2. ensure DashboardRestApi list_column and dashboard metadata are the same

Reply:
to ensure 2. we should add show_columns = list_columns on the REST API class, and follow the same existing pattern

Reply:
I will follow this PR, thanks, Daniel.